src/store/model-store.js

```diff
@@ -69,6 +70,15 @@ store.registerHandler('DELETE_ITEM', data => {
 });
 
 store.registerHandler('NEW_MODEL', data => {
+    assert.isString(data.name, 'The model name must be a string');
```

Comment:
It throws an error which is caught here and returned as a failed API response: https://github.com/TheFourFifths/consus/blob/1727b82d485ec602cb94ebc2d6054e3aa98c632b/src/api/model.js#L33

The response is tested here: https://github.com/TheFourFifths/consus/blob/1727b82d485ec602cb94ebc2d6054e3aa98c632b/test/functional/new-model.js#L65